debian/control

```diff
@@ -4,18 +4,23 @@ Priority: extra
 Maintainer: Sipwise Development Team <support@sipwise.com>
 Build-Depends: debhelper (>= 5),
                iptables-dev (>= 1.4),
-               libavcodec-dev,
-               libavfilter-dev,
-               libavformat-dev,
-               libavresample-dev,
-               libavutil-dev,
+               libavcodec-dev (>= 6:10),
+               libavcodec55 (>= 6:10) | libavcodec-dev (>= 6:11),
+               libavfilter-dev (>= 6:10),
+               libavfilter4 (>= 6:10) | libavfilter-dev (>=6:11),
+               libavformat-dev (>= 6:10),
+               libavformat55 (>= 6:10) | libavformat-dev (>=6:11),
+               libavresample-dev (>= 6:10),
+               libavutil-dev (>= 6:10),
                libcurl4-openssl-dev | libcurl4-gnutls-dev | libcurl3-openssl-dev | libcurl3-gnutls-dev,
                libevent-dev (>= 2.0),
                libglib2.0-dev (>= 2.30),
                libhiredis-dev,
+               libopus0 (>= 1.1),
```

Comment:
And this is also wrong, as that's an implementation detail of how libavcodec was linked. If libavcodec would be linked in the past or in the future against say libopus0.0 or libopus1, then even if you explicitly depend on that it would not mean it gets used at all when linking, and less so during run-time.